support-frontend/assets/pages/digital-subscription-checkout/components/thankYou/appsSection.jsx

```diff
@@ -86,7 +90,11 @@ const AppsSection = ({ countryGroupId }: PropTypes) => (
             nudgeIcon
             aria-label="Click to download the Guardian Daily app on Google Play"
             href={androidDailyUrl}
-            onClick={sendTrackingEventsOnClick('checkout_thankyou_daily_edition', 'DigitalPack', null)}
+            onClick={sendTrackingEventsOnClick({
```

Comment:
Are we happy not to distinguish between Android and iOS here?

Reply:
Good point, I agree it seems useful, I'll update to include this.